config/filesystems.php

```diff
@@ -38,7 +38,13 @@
         'public' => [
             'driver' => 'local',
             'root' => storage_path('app/public'),
-            'url' => env('APP_URL').'/storage',
+            'url' => env('APP_URL') . '/storage',
+            'visibility' => 'public',
+        ],
+
+        'uploads' => [
+            'driver' => 'local',
+            'root' => public_path('uploads'),
```

Comment:
bir şey yüklenecekse public dizinine yüklenmez. storage dizini altına yüklenir. public altına yüklenmesi pek doğru değildir. public diski kullanılır. php artisan storage:link ile storage/app/public dizini, public klasörü altına symlinklenir. https://laravel.com/docs/9.x/filesystem#the-public-disk